cipher/rijndael.c

```diff
@@ -1944,6 +2084,11 @@ selftest_fips_128 (int extended, selftest_report_func_t report)
 
   if (extended)
```

Comment:
Do we know if the extended is being called?

Reply:
Yes, extended is set to 1 when called from the global constructor:

```
static void GCC_ATTR_CONSTRUCTOR
_gcry_global_constructor (void)
{
  force_fips_mode = _gcry_fips_to_activate ();
  if (force_fips_mode)
    {
      no_secure_memory = 1;
      global_init ();
      _gcry_fips_run_selftests (1);
```

Reply:
The "1" here: _gcry_fips_run_selftests (1); is 'extended'.